tfx_bsl/public/beam/run_inference.py

```diff
@@ -21,19 +21,20 @@
 
 import apache_beam as beam
 import tensorflow as tf
+import pyarrow as pa
+from typing import Text, Optional
 from tfx_bsl.beam import run_inference
 from tfx_bsl.public.proto import model_spec_pb2
-from typing import Union
 from tensorflow_serving.apis import prediction_log_pb2
-
+from tensorflow_metadata.proto.v0 import schema_pb2
 
 @beam.ptransform_fn
-@beam.typehints.with_input_types(Union[tf.train.Example,
-                                       tf.train.SequenceExample])
+@beam.typehints.with_input_types(tf.train.Example)
 @beam.typehints.with_output_types(prediction_log_pb2.PredictionLog)
 def RunInference(  # pylint: disable=invalid-name
```

Comment:
Is the long term plan to deprecate the tf.example API? And only have a record batch API?

If so, mention it in a comment